src/ralph/back_office/tests/test_models.py

```diff
@@ -371,7 +372,7 @@ def test_return_report_when_user_not_assigned(self):
             _check_instances_for_transition([self.bo_asset], transition)
 
     @patch.object(ExternalService, "run")
-    def test_report_is_generated(self, mock_method):
+    def test_a_report_is_generated(self, mock_method):
```

Comment:
Most tests could be run in any order.
This test does not. Maybe It's worth to add comment that this test has to be run before other?